### PR TITLE
[Documentation] Update Korean documentation to English

### LINKED ISSUE #4: Update documents
- [x] README.md
- [ ] docs/TROUBLESHOOTING.md
- [ ] docs/USERGUIDE.md
- [x] docs/CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by hbinkim**
- translate documents written in Korean to English
- add trouble shooting guide
- add release note for v1.0.0
- edit necessary contents
- resolve #4

### CODE REVIEW

**Comment by woohhan** [comment]
Please convert to draft if currently working on, and assign reviewers/assignees/labels/projects

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
 
-![개발 워크플로우](docs/assets/workflow.png)
+![dev workflow](docs/assets/workflow_eng.png)
 
-1. 개발 환경을 생성합니다. Minikube none 드라이버로 rook을 포함해서 구축됩니다.
-2. 개발을 진행합니다. 개발 시 유닛, e2e 테스트를 포함해서 개발해야 합니다.
-3. 로컬 테스트를 진행합니다. dbox를 사용해서 테스트할 수 있습니다.
-4. 스태틱 테스트를 진행합니다.
-5. e2e 테스트를 진행합니다.
+1. Setup a local development environment by executing following command: `./tbox minikube`. It will install minikube with none driver option and deploy rook.
```
Comment (woohhan) [inline CONTRIBUTING.md:7]: We removed tbox, and added `testbox`. Some usage has changed.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,24 +1,20 @@
 # Contributing
 
-## 개발 워크플로우
+## Development Workflow
 
-![개발 워크플로우](docs/assets/workflow.png)
+![dev workflow](docs/assets/workflow_eng.png)
```
Comment (woohhan) [inline CONTRIBUTING.md:5]: Please delete original file

File: README.md [hunk 3]
```diff
@@ -1,16 +1,24 @@
...
-kubevirt-image-service(이하 kis)는 kubevirt에 통합되는 이미지, 볼륨 서비스를 제공합니다. (TODO: 데모)
 
-- `VirtualMachineImage`  : HTTP, S3, Local을 통해 외부에서 이미지를 임포트할 수 있습니다. (현재는 HTTP만 지원) 한 번 임포트된 이미지는 외부에서 다시 클론되지 않으며 내부에서 자체적으로 관리됩니다.
-- `VirtualMachineVolume` : 볼륨은 이미지로부터 생성할 수 있으며 CSI(Container Storage Interface)의 스냅샷-클론 기능을 활용하여 용량을 효율적으로 관리합니다.
-- `VirtualMachineExport` : 사용하던 볼륨을 qcow2 파일로 변환하여 외부에 전달할 수 있습니다.
+## What is kubevirt-image-service?
 
-## 시작하기
-다음 [유저 가이드](docs/USERGUIDE.md)를 참조하세요.
+kubevirt-image-service (KIS) is an integration with KubeVirt to provide image and volume services.
```
Comment (woohhan) [inline README.md:5]: please specify the link to the CDI project and add why we started this project and what's diff with CDI

**Comment by woohhan** [comment]
Please add/edit and delete the contents. Look at the big picture, not just translate into Korean. @hbinkim

File: README.md (hunk 3 above)
Comment (hbinkim) [reply README.md]: Done.

**Comment by woohhan** [comment]
LGTM! Thanks for handle this. @hbinkim

**Pull request merged**